### ISSUE TITLE
Support the option to expose the test case status immediately

### ISSUE THREAD

**Issue by mynguyenlgg** [inluxc/playwright-xray#94]
**Is your feature request related to a problem? Please describe.**
An option in config to expose the status of test case immediately instead of waiting for whole tests are competed. In case of running tests, we face the issue that leads to break the job, even about 70% test cases was completed.

**Describe the solution you'd like**
Either we could create an option in the configuration to do it, or we could allow the extension or implementation of the function to update on demand.

**Comment by inluxc** [comment]
So what you want is to the reporter submit in realtime the test results?

**Comment by mynguyenlgg** [comment]
> So what you want is to the reporter submit in realtime the test results?

Yes sure, actually we can do it in reality via Jira api to update the status ticket like Xray Jira, so I think we should do the same thing.

**Comment by Niklas-Back** [comment]
Yes it is possible to achieve and I don't think it requires to much coding. Perhaps we should put it as a feature request?

**Comment by mynguyenlgg** [comment]
Yes, thank you for your reply @Niklas-Back

**Comment by mstankovic** [comment]
upvote!
